src/SciMLOperators.jl

```diff
@@ -42,7 +45,9 @@ export ScalarOperator,
        MatrixOperator,
        DiagonalOperator,
        AffineOperator,
-       FunctionOperator
+       FunctionOperator,
+       TensorProductOperator,
+       ⊗
```

Comment:
I'm not sure about exporting this operator. Other than that, I think this is good to go.

Reply:
do you mean just the `\otimes` overload or TensorProductOperator?

Reply:
or both?

Reply:
either way i took out the export

Reply:
Just the `⊗`

Reply:
ok lemme add the TensorProductOperator export back

Reply:
done